embedded-service-registry/readme.md

```diff
@@ -0,0 +1,58 @@
+# Service registry in Azure Spring Apps
+
+This sample shows how to use embedded service registry in `Azure Spring Apps`.
+
+### Prerequisite
+
+* [JDK 11](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+
+### Concept
+Azure Spring Apps will create a dns record for each app, you can access to the app by http://{AppName}.
+You can use the url to communicate with other microservices, do not need to import any other dependencies.
+It also supports [blue-green deployment](https://learn.microsoft.com/en-us/azure/spring-apps/concepts-blue-green-deployment-strategies).
+
+Azure Spring Apps will probe each instance and put the health instance into the dns record. Following [health probe](https://learn.microsoft.com/en-us/azure/spring-apps/how-to-configure-health-probes-graceful-termination) you can set appropriate configuration.
+
```

Comment:
Shall we add some explanation about what's the behavior under blue-green deployment?